src/Cake.Npm.Tests/Version/NpmVersionToolTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Cake.Core;
+using Shouldly;
+using Xunit;
+
+namespace Cake.Npm.Tests.Version
+{
+    public class NpmVersionToolTests
+    {
+        public sealed class TheAutomatedInstallMethod
+        {
+            [Fact]
+            public void Should_Throw_If_Settings_Are_Null()
+            {
+                // Given
+                var fixture = new NpmVersionToolFixture();
+                fixture.Settings = null;
+
+                // When
+                var result = Record.Exception(() => fixture.Run());
+
+                // Then
+                result.IsArgumentNullException("settings");
+            }
+
+            [Fact]
+            public void Should_Add_Mandatory_Arguments()
+            {
+                // Given
+                var fixture = new NpmVersionToolFixture();
+
+                // When
+                var result = fixture.Run();
+
+                // Then
+                Assert.Equal("version", result.Args);
+            }
+
+            [Fact]
+            public void Should_Determine_Version_From_StandardOutput()
+            {
+                string[] versionInfo = new[]
+                {
+                    "{",
+                    "    npm: '5.8.0',",
+                    "    ares: '1.10.1-DEV',",
+                    "    cldr: '32.0',",
+                    "    http_parser: '2.8.0',",
+                    "    icu: '60.1',",
+                    "    modules: '57',",
+                    "    nghttp2: '1.25.0',",
+                    "    node: '8.11.1',",
+                    "    openssl: '1.0.2o',",
+                    "    tz: '2017c',",
+                    "    unicode: '10.0',",
+                    "    uv: '1.19.1',",
+                    "    v8: '6.2.414.50',",
+                    "    zlib: '1.2.11'",
+                    "}",
+"",
+"",
```

Comment:
Is the indentation here intentional?

Reply:
The indentation is intentional, to duplicate the slightly daft output from the `npm version` command.  Annoyingly, it returns a json fragment, unless you have an older version of npm and then it appends a block of text about updating that means we cant just treat the process output as json.